cypress/e2e/ManifestButton/PageDataExploration.cy.ts

```diff
@@ -0,0 +1,99 @@
+/// <reference types="cypress"/>
```

Comment:
peux-tu essayer, eventuellement d utiliser du code asynchrone? Si ca fonctionne ca ferait du code plus robuste `https://webtips.dev/webtips/cypress/async-await-in-cypress` je peux t aider si besoin.

Reply:
Je note et je regarde ça plus tard.